examples/step-20/doc/results.dox

```diff
@@ -10,16 +10,16 @@ the Raviart-Thomas element defines one degree per freedom per face and
 there are 72 faces parallel to the $x$-axis and the same number
 parallel to the $y$-axis):
 @code
-examples/\step-20> make run
-============================ Remaking Makefile.dep
-==============debug========= \step-20.cc
-============================ Linking \step-20
-============================ Running \step-20
+$ make run
+[ 66%] Built target step-20
+Scanning dependencies of target run
+[100%] Run step-20 with Release configuration
```

Comment:
The occurrence of `step-20` here may trigger our filter script to replace this with a link to step-20 (unnecessary, since we're already there) that unfortunately is not resolved by doxygen since we're inside `@code ... @endcode`. Can you escape it via a backslash as before?